app/editor/src/hooks/api-editor/interfaces/IContentModel.ts

```diff
@@ -29,7 +29,7 @@ export interface IContentModel extends IAuditColumnsModel {
   contentType?: IContentTypeModel;
   mediaTypeId: number;
   mediaType?: IMediaTypeModel;
-  licenseId: number;
+  licenseId?: number;
```

Comment:
This is definitely incorrect, this interface represents data from the API.  It will never be `null`, this is a required value.